pkg/ranger/content.go

```diff
@@ -109,7 +111,7 @@ func ServeContent(w http.ResponseWriter, r *http.Request, name string,
 		mw := multipart.NewWriter(pw)
 		w.Header().Set("Content-Type",
 			"multipart/byteranges; boundary="+mw.Boundary())
-		sendContent = func() io.Reader { return pr }
+		sendContent = func() io.ReadCloser { return pr }
```

Comment:
might make sense to wrap this with a NopCloser since closing pr was already handled by the line below

Reply:
Yes, I guess we should avoid the double close.